test/DataMCSpectraComparison/histosSimplified.py

```diff
@@ -13,9 +13,9 @@
 from SUSYBSMAnalysis.Zprime2muAnalysis.Zprime2muAnalysis_cff import goodDataFiltersMiniAOD
 
 process.source.fileNames =[#'file:./pat.root'
-'/TTTo2L2Nu_TuneCUETP8M2_ttHtranche3_13TeV-powheg-pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM'
+'/store/mc/RunIISummer16MiniAODv2/ZToMuMu_NNPDF30_13TeV-powheg_M_120_200/MINIAODSIM/PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/80000/824C363B-0AC8-E611-B4A5-20CF3027A580.root',
 			   ]
-process.maxEvents.input = -1
+process.maxEvents.input = 100 # Set to a reasonable number (e.g.100) when testing locally with cmsRun
```

Comment:
Back to 100 in the main code? I'd advise setting this back to -1, or at least remember doing that when it propagates from the development to the main branch.

Reply:
oups... missed it again... will fix